lib/registration.pm

```diff
@@ -138,7 +143,11 @@ sub fill_in_registration_data {
                     save_screenshot;
                 }
             }
-            send_key 'alt-n';
+            # this alt-n looks rather unmotivated. At least when we want
+            # to add addons during installation this presses alt-n the
+            # second time and skip the next addon selection screen which
+            # would break tests
+            send_key 'alt-n' if check_var('VIDEOMODE', 'text');
```

Comment:
This alt-n might work for 'next' after regcodes input.
I think it should be moved into above fill regcodes block if addons need codes.